src/PHPDebt/Console/Command/CodeSnifferConfigCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace PHPDebt\Console\Command;
+
+use PHP_CodeSniffer\Config;
+use PHP_CodeSniffer\Runner;
+use PHPDebt\PHPDebtMD;
+use SebastianBergmann\FinderFacade\FinderFacade;
+use SebastianBergmann\PHPLOC\Analyser;
+use Symfony\Component\Console\{
+	Command\Command,
+	Input\InputArgument,
+	Input\InputInterface,
+	Input\InputOption,
+	Output\OutputInterface
+};
+
+/**
+ * The class for all `phpdebt` command.
+ *
+ * @package PHPDebt\Console\Command
+ */
+class CodeSnifferConfigCommand extends Command {
+
+	/**
+	 * {@inheritdoc}
+	 */
+	protected function configure() {
+		$this->setName('phpdebt')
+			->addArgument('path', InputArgument::REQUIRED, 'The path of the file or directory that needs to be scanned.')
+			->addOption('standard', 's', InputOption::VALUE_REQUIRED, 'The path of the phpcs.xml.dist')
+			->setDescription('Enables option of utilizing project based PHPCS Config.');
```

Comment:
The description is for the command, not the option.

```
$ php src/index.php --help
Description:
  Enables option of utilizing project based PHPCS Config.
```

Reply:
Oh yes, I totally missed that. It's fixed now.